packages/default-reporter/src/reporterForClient/index.ts

```diff
@@ -88,6 +90,10 @@ export default function (
     reportUpdateCheck(log$.updateCheck, opts),
   ]
 
+  if (['install', 'update', 'add', 'remove'].includes(opts.cmd)) {
```

Comment:
```suggestion
const printExecutionTimeInCommands = ['install', 'update', 'add', 'remove']
  // something code...
  if (printExecutionTimeInCommands.includes(opts.cmd)) {
```

Would it be better to have them as an array of constants? It would be very convenient if there are other commands that need to execute time in the future (find/modify)

Reply:
done